app/models/miq_report/seeding.rb

```diff
@@ -84,12 +86,34 @@ def seed_record(path, report)
       end
     end
 
+    def template_type(path)
+      path.include?(RELATIVE_REPORT_DIR.to_s) ? "report" : "compare"
+    end
+
     def seed_files
+      seed_core_files + seed_plugin_files
+    end
+
+    def seed_core_files
       Dir.glob(REPORT_DIR.join("**/*.yaml")).sort + Dir.glob(COMPARE_DIR.join("**/*.yaml")).sort
     end
 
+    def seed_plugin_files
+      Vmdb::Plugins.flat_map do |plugin|
+        Dir.glob(plugin.root.join("#{RELATIVE_REPORT_DIR}/**/*.yaml")).sort + Dir.glob(plugin.root.join("#{RELATIVE_COMPARE_DIR}/**/*.yaml")).sort
```

Comment:
I want to avoid the "product" part in the plugin path...I've been trying to standardize on `/content/thing`, so in this case I'd expect for plugins `/content/reports` and `ontent/compare`.

Reply:
Hm okay, I assume we'd want to keep the current reports in `product/` and only the plugin reports would be in `content/`?

Reply:
Yeah for now they will stay in product in core.  Over time I'd like to move them to either a content directory in core, or a content directory in manageiq-content.

Reply:
> Hm okay, I assume we'd want to keep the current reports in product/ and only the plugin reports would be in content/?

@agrare My 2cents would be to move them in both.  Makes the code here easier to read, and just keeps things consistent between plugin and core.

Reply:
:+1:

Reply:
I'd rather not move the reports right now, particularly because reports are central to how the UI works and that's where they know them to exist.